core/runner/src/main/java/com/cognifide/aet/runner/processing/data/RunIndexWrappers/RunIndexWrapper.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * AET
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.runner.processing.data.RunIndexWrappers;
```

Comment:
Please fix package to contain only lowercase letter (this could be tricky).
Maybe it should be in `com.cognifide.aet.runner.processing.wrappers` ?